.devcontainer/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-12
+
+# The node image includes a non-root user with sudo access. Use the 
+# "remoteUser" property in devcontainer.json to use it. On Linux, update 
+# these values to ensure the container user's UID/GID matches your local values.
+# See https://aka.ms/vscode-remote/containers/non-root-user for details.
+ARG USERNAME=node
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# [Optional] Update UID/GID if needed
+RUN if [ "$USER_GID" != "1000" ] || [ "$USER_UID" != "1000" ]; then \
+        groupmod --gid $USER_GID $USERNAME \
+        && usermod --uid $USER_UID --gid $USER_GID $USERNAME \
+        && chmod -R $USER_UID:$USER_GID /home/$USERNAME; \
+    fi
+
```

Comment:
What if we do a default here, and have instructions to do other things like `npm run seed` and `npm run init` on first-run?  Not sure how this is integrated with CircleCI, but that can/should override `-e` in the CI/CD pipeline anyway.
  
```
RUN npm install

# Default dev entrypoint
ENTRYPOINT npm start
```

Reply:
CircleCI isn't currently using this Docker config, that would probably make sense as a separate issue.

WRT making `npm start` the default entrypoint, I don't think this makes a ton of sense if the initial image doesn't support it.  I think there's some design work to do here - making sure that `npm start` can work on a newly initialized container without taking away developer control of when databases are wiped or impacting other workflows like automated testing.  IMO this is another candidate for a new issue.